postgres/README.md

```diff
@@ -0,0 +1,223 @@
+## Example Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+To use HarbourBridge on a PostgreSQL database called mydb using pgdump output, run:
+
+```sh
+pg_dump mydb | harbourbridge -driver=pgdump
+```
+
+The tool can also be applied to an existing pg_dump file:
+
+```sh
+harbourbridge -driver=pgdump < my_pg_dump_file
+```
+
+To specify a particular Spanner instance to use, run:
+
+```sh
+pg_dump mydb | harbourbridge -driver=pgdump -instance my-spanner-instance
+```
+
+By default, HarbourBridge will generate a new Spanner database name to populate.
+You can override this and specify the database name to use by:
+
+```sh
+pg_dump mydb | harbourbridge -driver=pgdump -dbname my-spanner-database-name
+```
+
+HarbourBridge generates a report file, a schema file, and a bad-data file (if
```

Comment:
we can write this section in common file, where information about -prefix flag is written

Reply:
Then it would a very large section containing redundant commands i.e. pgdump and mysqldump. We have already provided sample usage command in common file. This is in more detail so I thought of putting it in respective package readme file.
Still let me know what do you think!

Reply:
not the commands, just this 3 lines:
HarbourBridge generates a report file, a schema file, and a bad-data file (if
there are bad-data rows). You can control where these files are written by
specifying a file prefix.

anyways, its not a huge problem, if you think its better here, fine with me

Reply:
oh ok. I thought you are talking about while `example-usage` section.